op-mode-definitions/tinc.xml.in

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="show">
+    <children>
+      <node name="interfaces">
+        <children>
+          <node name="tinc">
+            <properties>
+              <help>Show Tinc VPN interface information</help>
+            </properties>
+            <children>
+              <leafNode name="detail">
+                <properties>
+                  <help>Show detailed Tinc VPN interface information</help>
+                </properties>
+                <command>${vyos_op_scripts_dir}/show_interfaces.py --intf-type=tinc --action=show</command>
+              </leafNode>
+            </children>
+          </node>
+          <tagNode name="tinc">
+            <properties>
+              <help>Show Tinc VPN interface information</help>
+              <completionHelp>
+                <script>sudo ${vyos_completion_dir}/list_interfaces.py --type tinc</script>
+              </completionHelp>
+            </properties>
+            <command>${vyos_op_scripts_dir}/show_interfaces.py --intf=$4</command>
+            <children>
+              <leafNode name="brief">
+                <properties>
+                  <help>Show summary of specified Tinc VPN interface information</help>
+                </properties>
+                <command>${vyos_op_scripts_dir}/show_interfaces.py --intf="$4" --action=show-brief</command>
+              </leafNode>
+            </children>
+          </tagNode>
+        </children>
+      </node>
+    </children>
+  </node>
+  <node name="generate">
+    <children>
+      <tagNode name="tinc">
+        <properties>
+          <help>Generate tinc vpn tunnel configuration</help>
+          <completionHelp>
+            <script>sudo ${vyos_completion_dir}/list_interfaces.py --type tinc</script>
+          </completionHelp>
+        </properties>
+        <children>
+          <leafNode name="host-conf">
+            <properties>
+              <help>Generate tinc vpn tunnel host configuration path</help>
+            </properties>
+            <command>scp "$5" "/etc/tinc/$3/hosts/"</command>
```

Comment:
I don't think any commands should implicitly call `scp`. All generate commands we have output the configuration to stdout, tinc shouldn't be an exception. The user can then decide how to use that generated config, whether to scp it, copy and paste, or send to someone.

Also, simply showing the config file from the local side isn't very useful. The user can always `cat` it by hand, so it doesn't add any value. A proper generate command should produce a config that the remote side can use unchanged.

Reply:
done

 I removed this command